src/index.js

```diff
@@ -92,7 +93,7 @@ app.get("/users/:username", async (req, res) => {
         const retval = {
             apps: {
                 recentlyAdded: await recentlyAddedData(db, username, limit),
-                public: await publicAppsData(db, limit),
+                protected: await publicAppsData(db, username, limit),
```

Comment:
I'm confused about why this says `protected` now, not sure if that's an intended change.

Reply:
That was unintended. I'm not sure why that changed either. 😕 Thanks for catching that.